### DISCUSSION TITLE
[Feature Request] Hide shortcode from the preview in the main page

### DISCUSSION THREAD

**Discussion by luigi311** [chipzoller/hugo-clarity#224] [Q&A]
If the post consists of mostly shortcode for things such as chartjs then the preview of the post will display shortcode in it. This should be hidden as its not something the end user sees or wishes to see.

This is what it looks like now
![image](https://user-images.githubusercontent.com/4695527/121759391-bb604400-cae2-11eb-8eae-a4ac452ee8ab.png)

This is what it should look like
![image](https://user-images.githubusercontent.com/4695527/121759464-05e1c080-cae3-11eb-9739-c7ac9e93c5b3.png)

**Comment by luigi311** [comment]
Source code can be found here https://gitlab.com/Luigi311/finance.luigi311.com for additional information if needed.

**Comment by chipzoller** [comment]
We also need to ignore tips if they occur in the first few sentences.

**Comment by chipzoller** [comment]
@onweru I think we should fast track this one.

**Comment by onweru** [comment]
Turns out the solutions to this problem is [explicitly defining your summary](https://discourse.gohugo.io/t/exclude-image-shortcode-from-summary/31181/2). This theme already supports that via either `summary` or `abstract` from the frontmatter.
